nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -192,6 +192,10 @@ let
 
       server_tokens ${if cfg.serverTokens then "on" else "off"};
 
+      ${optionalString (cfg.proxyCache.enable) ''
+        proxy_cache_path /var/cache/nginx keys_zone=${cfg.proxyCache.keysZoneName}:${cfg.proxyCache.keysZoneSize} levels=${cfg.proxyCache.levels} use_temp_path=${if cfg.proxyCache.useTempPath then "on" else "off"} ${optionalString (cfg.proxyCache.inactive != null) "inactive=${cfg.proxyCache.inactive}"} ${optionalString (cfg.proxyCache.maxSize != null) "max_size=${cfg.proxyCache.maxSize}"};
```

Comment:
Please split this line into multiple lines.

Reply:
If split lines, then in end the nginx configuration will display like this:
```
  proxy_cache_path /var/cache/nginx/proxy_cache levels=1:2
  keys_zone=CACHE:32m max_size=1g inactive=60m;
  proxy_ignore_headers "Set-Cookie";
  server {
    listen 0.0.0.0:80 ;
    listen [::0]:80 ;
```
Difficult to view configuration.
Can't generate this variant:
```
  proxy_cache_path /var/cache/nginx/proxy_cache levels=1:2
                                keys_zone=CACHE:32m max_size=1g inactive=60m;
  proxy_ignore_headers "Set-Cookie";
  server {
    listen 0.0.0.0:80 ;
    listen [::0]:80 ;
```